SoftClip.h

```diff
@@ -0,0 +1,50 @@
+#ifndef SOFT_CLIP_H
+#define SOFT_CLIP_H
+
+#include "AudioOutput.h"
+
+template<unsigned int SMOOTHING, typename T = AudioOutputStorage_t>
```

Comment:
I did that so that the lookup table would be statically allocated. If the lookup is the right way to go, then if this is not a template parameter, I do not see the way to statically allocate it without a template but maybe you can enlighten me on that?